src/components/common/RequestStatusBox.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Grid, Responsive } from 'semantic-ui-react'
+import styled from 'styled-components'
+
+const RequestStatusBoxGrid = styled(Grid)`
```

Comment:
Do we want to start adding docs for reusable components?

Reply:
What would you put in docs for this component?

Reply:
What it does and maybe a usage example (if we are relatively sure it's not going to change radically).

Reply:
It is hard to keep non-running usage examples up to date TBH. Tests prob work better for that purpose.

> if we are relatively sure it's not going to change radically

not sure it's going to survive introduction of design system